src/main/java/fi/metatavu/metaform/server/metaforms/ReplyController.java

```diff
@@ -44,6 +58,11 @@
 @ApplicationScoped
 public class ReplyController {
 
+  private static MetaformTableColumnType[] SUPPORTED_TABLE_COLUMN_TYPES = new MetaformTableColumnType[] {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "SUPPORTED_TABLE_COLUMN_TYPES" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS3008))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=60&issues=AWY4b1pzvOWp7Yd6rqjj&open=AWY4b1pzvOWp7Yd6rqjj)

[ISSUE_KEY]: AWY4b1pzvOWp7Yd6rqjj